### ISSUE TITLE
Consolidate info about hard-coded paths

### ISSUE THREAD

**Issue by andy5995** [AppImage/docs.appimage.org#84]
There seems to be two different sections where hard-coded paths are mentioned. I suggest it might be better to consolidate and merge the information. I'll do this if you'd like, but I need to know how you would like it consolidated.

Maybe move the info from here and merge it onto the reference page.  Then add a link to the reference page?
https://docs.appimage.org/packaging-guide/manual.html#no-hard-coded-paths

https://docs.appimage.org/reference/best-practices.html#binaries-must-not-use-compiled-in-absolute-paths

**Comment by TheAssassin** [comment]
Good point. Since this is a best practice that applies to all applications, the contents should be merged into the best practices page. We should link more often to this page anyway.
